src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -30,20 +30,21 @@ import { heading_to_string, radiansToDegrees, degreesToRadians } from '../utilit
  * @class AircraftCommander
  */
 export default class AircraftCommander {
-    constructor(aircraftController, onChangeTransponderCode) {
+    constructor(aircraftController) {
```

Comment:
hmm, I'm torn.  should we pass an entire instance of the `AircraftController` or just references to the two methods we need a few lines below?  what do you think?

Reply:
@n8rzz my personal preference would be to pass in the 2 methods. it would also make the fixture for the test a bit simpler

Reply:
yeah, unless @erikquinn has any concerns, let's do that.  Its way cleaner that way.

Reply:
@n8rzz this change is in and awaiting review

Reply:
It was being passed in order to do the binding. It makes sense to me to bind here rather than the aircraftController, because the reason is obvious here, but not obvious in the aircraftController.

But if everything works doing it the other way, I'm not opposed.

Reply:
@n8rzz if I do the binding in the controller, is it acceptable to do it as part of the call to the `Commander` constructor? Or does it need to go into `_setupHandlers()` and then the creation of the commander would need to be after that.

`this._aircraftCommander = new AircraftCommander(this.onRequestToChangeTransponderCode.bind(this), this.findAircraftById.bind(this));`

Reply:
It's not a handler used by this class, so it should be fine just like that. Seems more sensible in the aircraftController than I thought it would be, so I'm fully satisfied with that strategy after verifying that it works :smiley:

Reply:
@erikquinn I made this last change, let me know if there is anything else.